src/views/refund/steps.js

```diff
@@ -28,19 +29,24 @@ const stepOneStyles = theme => ({
   },
 });
 
-const StyledStepOne = ({ classes }) => (
+const StyledStepOne = ({ classes, setRefundFile, setTransactionHash }) => (
   <View className={classes.wrapper}>
-    <DropZone className={classes.dropZone}>
+    <DropZone className={classes.dropZone} setRefundFile={setRefundFile}>
       <p className={classes.info}>Drag the Refund JSON file here</p>
       <span className={classes.info}>or</span>
       {/*TODO: add ability to upload*/}
       <FileUpload text={'Select file'} />
     </DropZone>
+    {/*FIXME: HOW IN THE NAME OF FUCKING GOD CAN I PUT THIS UNDERNEATH THE DROPZONE */}
```

Comment:
LOL, I will have a look in a second :)

Reply:
Here is a nice reference for [flex-box](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#article-header-id-3)